sdk/servicebus/service-bus/tests.yml

```diff
@@ -28,4 +28,3 @@ jobs:
         AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AAD_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         SERVICEBUS_CONNECTION_STRING: $(SERVICEBUS_CONNECTION_STRING)
-        SERVICEBUS_CONNECTION_STRING_BROWSER: $(SERVICEBUS_CONNECTION_STRING_BROWSER)
```

Comment:
@ramya0820 I understood https://github.com/Azure/azure-sdk-for-js/pull/6628#discussion_r367752444 to mean that we don't even need to have the `SERVICEBUS_CONNECTION_STRING` mapping in the tests.yml file

Reply:
cc: @danieljurek ? Looking at Event Hubs test.yml, that doesn't seem to be the case?